lib/etlStore.js

```diff
@@ -0,0 +1,135 @@
+const log = require('./utils/logger');
```

Comment:
The `pharmacy-data-etl` has the recent improvements in. Perhaps take that file as a basis?

Reply:
I brought the changes across, I've removed a few things though.
The lastRunDate is related to the ETL and not used within the toolkit, so that has been pushed up to the ETL file in the shis data ETL branch I'm verifying this against.

I've removed the tracking of the modifiedIDs and its display in the summary for now as this is not used in the shis ETL. This should be revisited when the pharmacy-data-etl is updated to use the toolkit. 

I think better use could be made of tracking a modified ID list in the toolkit. At the moment the pharmacy-data-etl deletes modified records, then relies on the ETL refreshing missing records. The downside with this is if there's an error updating the record after deletion it won't be replaced. It's probably better to have out of date data than no data.

Reply:
Good point about errors during updates on modified records. Ofc, we know that would never happen!
Perhaps they should only be removed once the new data has been acquired. That would work when the modified list doesn't contain records that have been removed, as is the case for syndication.